Milyli.ScriptRunner.Services.Interfaces/V1/Models/Responses/ScriptRunResponse.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Milyli.ScriptRunner.Services.Interfaces.V1.Models.Responses
+{
+	using System.Collections.Generic;
+	using Models;
+
+	/// <summary>
+	/// Base response to Script Run requests
+	/// </summary>
+	public class ScriptRunResponse
+	{
+		public ScriptRun ScriptRun { get; set; }
```

Comment:
It looks like there are still some of the models without XMLDocs

- [ ] Make sure everything has XMLDocs